api/spec/controllers/mno_enterprise/jpi/v1/admin/invoices_controller_spec.rb

```diff
@@ -2,186 +2,253 @@
-        end
-      end
+      it { expect(data['id']).to eq(bill.id) }
+      it { expect(data['invoice']['total_due']['fractional']).to eq(invoice.total_due.cents.to_f.to_s) }
+      it { expect(data['invoice']['price']['fractional']).to eq(invoice.price.cents.to_f.to_s) }
     end
 
-    describe 'GET #last_portfolio_amount' do
-      subject { get :last_portfolio_amount }
-
-      it_behaves_like 'a jpi v1 admin action'
-
-      context 'success' do
-        before { subject }
+    describe 'DELETE #delete_adjustment' do
+      subject { delete :delete_adjustment, id: invoice.id, bill_id: bill.id }
 
-        it { expect(response).to be_success }
+      let(:data) { JSON.parse(response.body) }
 
-        it 'returns a valid amount' do
-          expected = {'last_portfolio_amount' => {'amount' => tenant.last_portfolio_amount.amount, 'currency' => tenant.last_portfolio_amount.currency_as_string}}
-
-          expect(response.body).to eq(expected.to_json)
-        end
+      before do
+        stub_api_v2(:get, "/bills", bill, [],
+          {
+            fields: { bills: 'id' },
+            filter: { 'adjustment' => 'true', 'invoice.id' => invoice.id, 'id' => bill.id },
+            page: { number: 1, size: 1}
+          })
       end
+      before { stub_api_v2(:delete, "/bills/#{bill.id}") }
+      before { stub_api_v2(:get, "/invoices/#{invoice.id}", invoice, [], { fields: { invoices: 'price,total_due' } }) }
+      before { subject }
+
+      it { expect(data['invoice']['total_due']['fractional']).to eq(invoice.total_due.cents.to_f.to_s) }
+      it { expect(data['invoice']['price']['fractional']).to eq(invoice.price.cents.to_f.to_s) }
     end
 
-    describe 'GET #last_commission_amount' do
-      subject { get :last_commission_amount }
+    describe 'POST #send_to_customer' do
+      subject { post :send_to_customer, id: invoice.id }
 
-      it_behaves_like 'a jpi v1 admin action'
+      let(:data) { JSON.parse(response.body) }
 
-      context 'success' do
-        before { subject }
-        let(:last_commission_amount) { {'last_commission_amount' => {"amount"=>"4123.45", "currency"=>"AUD"}} }
+      before { allow(invoice).to receive(:organization).and_return(organization) }
+      before { stub_api_v2(:get, "/invoices/#{invoice.id}", invoice, %i(organization), { fields: { invoices: 'organization', organizations: 'id' } }) }
+      before { stub_api_v2(:get, "/organizations/#{organization.id}", organization, %i(orga_relations), { fields: { organizations: 'orga_relations',  orga_relations: 'id,user_id,role' } }) }
+      before { subject }
 
-        it 'returns the sum of commissions' do
-          expect(response).to be_success
-          expect(JSON.parse(response.body)).to eq(JSON.parse(last_commission_amount.to_json))
-        end
-      end
+      it { expect(data['status']).to eq('request_sent') }
     end
+
+  # TODO: re-spec reporting endpoints
```

Comment:
Could you use `xdescribe` instead of commenting?
This way the specs show as pending instead of being ignored